.dockerignore

```diff
@@ -1,2 +1,2 @@
 bin
-.buildstate
+.tmp
```

Comment:
nit: why is this added here instead of just doing `mkdir -p` in the Dockerfile

Reply:
Actually the directory is created in Dockerfile and this line is unneeded for `.dockerignore`, but just added for consistency with `.gitignore`.